model/generate.xml

```diff
@@ -312,7 +313,8 @@
 
   <command symbol="fetch-history" output="history_row" category="ONLINE" network="true" description="Get list of output points, values, and spends for a payment address. Requires a Libbitcoin server connection.">
     <option name="format" type="encoding" description="The output format. Options are 'info', 'json' and 'xml', defaults to 'info'." />
-    <argument name="PAYMENT_ADDRESS" stdin="true" type="payment_address" description="The payment address. If not specified the address is read from STDIN." />
```

Comment:
This compatibility break on history/balance should be mostly corrected from the last push (-a is no longer required for an address), but the remaining break is that the commands can no longer accept a payment address from stdin (Or at least I couldn't figure out how to get that to work as an either/or with the script_hash option).  As it is, it then required an extra check to make sure at least one argument is specified and valid.  Any alternate suggestions?

I'm holding off on updating watch-address for the moment since `client` doesn't yet support it, and I'd have to review what the server is expecting as well (it likely needs an update also).  It's been on my list of things to review/update before v4 though.